dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestLogger.kt

```diff
@@ -94,6 +95,20 @@ class DataRequestLogger {
         bulkDataRequestLogger.info(logMessage)
     }
 
+    /**
+     * Logs an appropriate message when a data request has been stored in the database.
+     */
+    fun logBulkDataOverwiew(
+        r: BulkDataRequestResponse,
```

Comment:
Single letter variables are only accpetable in loops. Choose sometinh more verbose.